content/api/storage/openapi/_index.nb.md

```diff
@@ -0,0 +1,8 @@
+---
+title: OpenAPI spesifikasjon for lagringskomponenten
```

Comment:
Storage er jo navnet på komponenten, så det skader vel ikke å bruke storage-komponenten og tilsvarende for event-komponenten der dette skal oversettes? Vil kanskje være mer inutitivt for den som leser ?